saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/classifier/SparseNetworkLBP.scala

```diff
@@ -5,20 +5,20 @@ import edu.illinois.cs.cogcomp.lbjava.learn.{ LinearThresholdUnit, SparseNetwork
 /** Created by Parisa on 5/24/15.
   */
 class SparseNetworkLBP() extends SparseNetworkLearner {
-  var net = network
-  var iConjuctiveLables = conjunctiveLabels
-  override def getLTU(i: Int): LinearThresholdUnit = {
-    var a: LinearThresholdUnit = net.get(i).asInstanceOf[LinearThresholdUnit]
-    a
-  }
-  def getnumExamples: Int = {
-    numExamples
-  }
-  def getnumFeatures: Int = {
-    numFeatures
-  }
-  def getbaseLTU: LinearThresholdUnit = {
-    baseLTU
-  }
 
+  /** Pseudo-Getter method for [[conjunctiveLabels]] field.
+    * @return
+    */
+  def iConjuctiveLables: Boolean = this.conjunctiveLabels
+
+  /** Setter method for [[conjunctiveLabels]] field.
+    * @param newVal
+    */
+  def iConjuctiveLables_=(newVal: Boolean): Unit = this.conjunctiveLabels = newVal
+
+  /** Override method to return LTU of type [[LinearThresholdUnit]].
+    * @param i Position of the LTU unit in the network.
+    * @return LTU unit of type [[LinearThresholdUnit]]
+    */
+  override def getLTU(i: Int): LinearThresholdUnit = super.getLTU(i).asInstanceOf[LinearThresholdUnit]
 }
```

Comment:
So that boolean does not have a public setter, hence we'll have to override it for now. Will see if it would make sense to move the weight update logic inside the LBJava SparseNetworkLearner class and call it directly.